modules/promotions/admin-menu-items/base-promotion-item.php

```diff
@@ -33,17 +33,31 @@ public function get_image_url() {
 	}
 
 	public function render() {
+		$config = [
+			'title' => $this->get_promotion_title(),
+			'description' => $this->get_promotion_description(),
```

Comment:
Signature of `render_promotion_description():void` changed to `get_promotion_description():string`. The base change affects several other classes that depend on this base and are now added to this PR.

Reply:
@Barcov I don't think we can delete `render_promotion_description()` as it's a public function.

I suggest:

Adding a new one for:

```
public function get_promotion_description() {
		return esc_html__(
			'Custom Code is a tool gives you one place where you can insert scripts, rather than dealing with dozens of different plugins and deal with code.',
			'elementor' );
}
```

And modifying:

```
public function render_promotion_description() {
		echo $this->get_promotion_description()
}
```